pkgs/development/python-modules/tokenizers/default.nix

```diff
@@ -72,6 +74,10 @@ in buildPythonPackage rec {
     rust.rustc
   ]);
 
+  buildInputs = lib.optionals stdenv.isDarwin [
```

Comment:
```suggestion
  buildInputs = [
```

Reply:
I would argue that it doesn't really save on complexity as there's implicit mental context needed to know that it's null.

Also, for other platforms (e.g. freebsd); I think the original patch is more aligned with the intended behavior.

Reply:
There are pros and cons to both options. I went with what I saw when grep'ing "libiconv". We can refactor things once there is a clear guidance (e.g. from discussions such as #122416)